components/EpisodeDetails/EpisodeDetails.styled.js

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components";
+import Image from "next/image";
+
+export const EpisodeDetailsArticle = styled.article`
+  position: relative;
```

Comment:
position relative is default, you dont need it here

Reply:
Thanks, nice hint! Forgot to remove it, I had a button in there before with `position: absolute`.